src/components/molecules/InboxItem.vue

```diff
@@ -5,7 +5,7 @@
       'flex items-center w-full h-16 md:h-20 border-2 border-transparent focus:border-black hover:bg-gray-100 cursor-pointer',
     ]"
   >
-    <div class="p-4">
+    <div class="p-2 md:p-4">
```

Comment:
We can get rid of this div and apply the styling on the p tag. instead of padding it will become margin instead. so: m-2 md:m-4

Reply:
It's a good suggestion but the `<p>` tag isn't responsive enough. Regardless this p tag is being switched out for an icon soon

Reply:
ok sounds good